nevergrad/optimization/base.py

```diff
@@ -246,6 +247,23 @@ def remove_all_callbacks(self) -> None:
         """
         self._callbacks = {}
 
+    def request(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Potential other names: `suggest` and `propose`.
`request` seems better to me because it highlights the fact that this will be actually *enforced* at the next `ask`.

Reply:
Hum the problem is that request is less intuitive I think.
I would prefer suggest, you can still write in the doc that the optimizer WILL use the candidate

Reply:
Hum the problem is that request is less intuitive I think.
I would prefer suggest, you can still write in the doc that the optimizer WILL use the candidate

Reply:
`suggest` used to be the old name of `ask`... so I am somehow uncomfortable with it :s

Reply:
Discussion summary: `suggest` is more intuitive